### ISSUE TITLE
Crashes when using multiScope

### ISSUE THREAD

**Issue by nice6599** [j4s0n-c/trowaSoft-VCV#24]
rack keeps crashing when using multiScope

I installed the plugin through the plugin manager 

## Version

Version 0.6.0

## Hardware

this is also in the crash log
```
Model: MacBookPro14,3, BootROM MBP143.0169.B00, 4 processors, Intel Core i7, 2.8 GHz, 16 GB, SMC 2.45f0
Graphics: Radeon Pro 555, Radeon Pro 555, PCIe, 2 GB
Graphics: Intel HD Graphics 630, Intel HD Graphics 630, Built-In
Memory Module: BANK 0/DIMM0, 8 GB, LPDDR3, 2133 MHz, 0x80CE, 0x4B3445424533303445422D45474347202020
Memory Module: BANK 1/DIMM0, 8 GB, LPDDR3, 2133 MHz, 0x80CE, 0x4B3445424533303445422D45474347202020
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x173), Broadcom BCM43xx 1.0 (7.77.37.5.1a3)
Bluetooth: Version 6.0.2f2, 3 services, 18 devices, 1 incoming serial ports
Network Service: Wi-Fi, AirPort, en0
USB Device: USB 3.0 Bus
USB Device: iBridge
Thunderbolt Bus: MacBook Pro, Apple Inc., 23.1
Thunderbolt Bus: MacBook Pro, Apple Inc., 23.1
```

## Log
macOS Crash log
[LOG.txt](https://github.com/VCVRack/Rack/files/1875495/LOG.txt)

VCV Rack Log
[log.txt](https://github.com/VCVRack/Rack/files/1875503/log.txt)

**Comment by j4s0n-c** [comment]
Thanks for your detailed feedback. It's just the new version with effects and background color. We don't have the time right now to figure out what is different about mac and why it crashes. Windows and linux builds work fine.
So, we just made some changes that will give a working (old version) multiScope on mac and the new features will only build for windows and linux for now. And, we finally got rid of the pastel colors.
